adapter_test.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+from adapter.adapter import CephAdapter, CephAdapterError
+import warnings
+
+
+# For testing ceph interaction only.
+# Not for testing unittest
+class CephTest(unittest.TestCase):
+
+    def setUp(self):
+        warnings.filterwarnings(action="ignore", message="unclosed", category=ResourceWarning)
+    def tearDown(self):
+        if self.adpt:
+            self.adpt.remove("testParcel")
+        
+    def test_order(self):
+
+        test_data_name = "testParcel"
+        test_data_bytes = bytes([10, 20])
+
+        print("-- connect start")
+        self.adpt = CephAdapter()
+        self.adpt.connect("172.105.221.117", 7480, "adapter/key.json", "amo")
```

Comment:
Well... This does not seem to be a "unit" test.

Reply:
Actually, adapter_test.py file is just for debugging for me. I'll remove it from git repository. Actual API server's unit test will be performed on auth/tests.py.